googlemock/test/gmock-matchers_test.cc

```diff
@@ -2778,7 +2778,7 @@ class FloatingPointTest : public testing::Test {
 
   // Pre-calculated numbers to be used by the tests.
 
-  const size_t max_ulps_;
+  const unsigned int max_ulps_;
```

Comment:
max_ulps_ is initialized from a size_t in Floating<FltType>.
We should just be mirroring that type here.